ReactTypings/react-native.d.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path=".vscode/typings/react/react.d.ts" />
+/// <reference path=".vscode/typings/react/react-global.d.ts" />
+/// <reference path=".vscode/typings/react/react-dom.d.ts" />

```

Comment:
Actually, we shouldn't reference `react-dom.d.ts` since that doesn't apply to react-native